pkg/pgclient/config.go

```diff
@@ -17,13 +17,14 @@ import (
 	"github.com/timescale/promscale/pkg/limits"
 	"github.com/timescale/promscale/pkg/log"
 	"github.com/timescale/promscale/pkg/pgmodel/cache"
-	"github.com/timescale/promscale/pkg/pgmodel/ingestor/trace"
+	"github.com/timescale/promscale/pkg/pgmodel/ingestor"
 	"github.com/timescale/promscale/pkg/version"
 )
 
 // Config for the database.
 type Config struct {
 	CacheConfig             cache.Config
+	IngestorFlags           ingestor.Config
```

Comment:
nit:

```suggestion
	IngestorConfig           ingestor.Config
```

For consistency `Config` is used to refer to almost every other configuration grouping, and we already have the `CacheConfig` above.